### DISCUSSION TITLE
GestureHandlerRootView Docs Clarification

### DISCUSSION THREAD

**Discussion by scousino** [software-mansion/react-native-gesture-handler#1627] [Q&A]
Hi, I was hoping for some clarification on how to utilize the gesture handler in a component library. In the documentation it says:

> If you're using gesture handler in your component library, you may want to wrap your library's code in the GestureHandlerRootView component. This will avoid extra configuration for the user.

 What does it mean by "library's code"? This is too vague. If I'm building a UI component library, does every component need to be wrapped in their own GestureHandlerRootView, or am I supposed to wrap some sort of entry point?

**Comment by pierroo** [comment]
Exactly!
Did you find an answer to your question yet (one year later^^) perhaps @scousino ?
